lib/gollum/views/latest_changes.rb

```diff
@@ -10,6 +10,7 @@ def title
 
       def versions
         i = @versions.size + 1
+        url_start_pos = @page_dir.to_s.empty? ? 0 : Pathname.new(@page_dir).cleanpath.to_s.length+1
```

Comment:
Could you try to make this a little more legible? Difficult to understand what is going on here.